modules/runner/src/integrationTest/java/org/apache/ignite/internal/rebalance/ItRebalanceDistributedTest.java

```diff
@@ -583,6 +585,59 @@ void testRebalanceWithTheSameNodes() throws Exception {
         verifyThatRaftNodesAndReplicasWereStartedOnlyOnce();
     }
 
+    @Test
+    void testRaftClientsUpdatesAfterRebalance() throws InterruptedException, ExecutionException, TimeoutException {
+        Node node = getNode(0);
+
+        createZone(node, ZONE_NAME, 1, 1);
+
+        createTable(node, ZONE_NAME, TABLE_NAME);
+
+        assertTrue(waitForCondition(() -> getPartitionClusterNodes(node, 0).size() == 1, AWAIT_TIMEOUT_MILLIS));
+
+        Set<Assignment> assignedNodeBeforeRebalance = getPartitionClusterNodes(node, 0);
```

Comment:
Minor. It's probably a typo. assigned -> assignments in assignedNodeBeforeRebalance